cost-analyzer/templates/NOTES.txt

```diff
@@ -2,10 +2,18 @@
 
 --------------------------------------------------
 {{- $servicePort := .Values.service.port | default 9090 -}}
-Kubecost has been successfully installed. When pods are Ready, you can enable port-forwarding with the following command:
-    
-    kubectl port-forward --namespace {{ .Release.Namespace }} deployment/{{ template "cost-analyzer.fullname" . }} {{ $servicePort }}
-    
-Next, navigate to http://localhost:{{ $servicePort }} in a web browser.
+Kubecost has been successfully installed.
+
+Please allow 5-10 minutes for Kubecost to gather metrics and report accurate health status in the UI and data to appear.
+
+If you have configured cloud-integrations, it can take up to 48 hours for cost reconciliation to occur.
+
+When using Durable storage (Enterprise Edition) please allow up to 4 hours for data to be collected and the UI to be healthy.
+
+When pods are Ready, you can enable port-forwarding with the following command:
+
+    kubectl port-forward --namespace kubecost deployment/kubecost-cost-analyzer 9090
+
+Next, navigate to http://localhost:9090 in a web browser.
 
 Having installation issues? View our Troubleshooting Guide at http://docs.kubecost.com/troubleshoot-install
```

Comment:
We might consider updating this to point to guide.kubecost.com. It's not mandatory as we have redirects in place.
i.e. https://guide.kubecost.com/hc/en-us/articles/4407601830679-Troubleshoot-Install

Reply:
Any benefit to point to guide.kubecost? These urls are far less flexible IMO in that they have the article ID hardcoded in them.

Reply:
Agree with web, especially if we ever migrate from zd.